### PR TITLE
create thanks and early career pages, eliminate student editors.

### PR DESCRIPTION
**Pull request by mrshirts**
Changes proposed in the student editor PR.

### CODE REVIEW

File: _about_livecoms/02_thanks.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+excerpt: Thanks for special services in helping make LiveCoMS possible
+permalink: /about/thanks/
+---
+
+LiveCoMS wishes to give special thanks to:
+
+* CU Boulder Libraries, for their generous donation of seed money to start the journal.
+
+* The CU Boulder Department of Chemical and Biological Engineering for
+  agreeing to help serve as publisher, and providing logistical support
```
Comment (dmzuckerman) [inline _about_livecoms/02_thanks.md]: @mrshirts is your dept the publisher?  I would have thought it's just CU.  If so, just break this into two bullets

File: _about_livecoms/03_earlycareer.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+layout: single
+sidebar:
+  nav: earlycareer.md
+title: For Early Career Scientists 
+excerpt: How early career scientists can get involved with LiveCoMS
+permalink: /about/earlycareer/
+---
+
+We welcome early career scientists (undergraduate students, graduate
+students and postdocs) to help participate in LiveCoMS.  There are two
```
Comment (dmzuckerman) [inline _about_livecoms/03_earlycareer.md]: "to help participate" should probably be "to participate"

File: _about_livecoms/03_earlycareer.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+layout: single
+sidebar:
+  nav: earlycareer.md
+title: For Early Career Scientists
```
Comment (dmzuckerman) [inline _about_livecoms/03_earlycareer.md]: I wonder whether students will realize this is for them.  What about "For sudents and early-career scientists"

File: _about_livecoms/03_earlycareer.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+excerpt: How early career scientists can get involved with LiveCoMS
+permalink: /about/earlycareer/
+---
+
+We welcome early career scientists (undergraduate students, graduate
+students and postdocs) to help participate in LiveCoMS.  There are two
+ways you can participate:
+
+* If you are a graduate student or postdoc, we encourage you to sign
+  up to be a student reviewer. Please email
```
Comment (dmzuckerman) [inline _about_livecoms/03_earlycareer.md]: I would insert a sentence here: "The *requirement* for a student reviewer for every submission is a unique strength of LiveCoMS that should enhance the pedagogical value of every accepted paper."

File: _about_livecoms/02_thanks.md [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+  for journal finances.
+
+* CU Boulder CEAS Dean Bobby Braun and CU Boulder Provost Russell More
+  for their support to have CU Boulder serve as publisher of LiveCoMS.
```
Comment (dmzuckerman) [inline _about_livecoms/02_thanks.md]: Here you also thank CU for publisher role - probably this is the place to have it.

File: _about_livecoms/02_thanks.md (hunk 1 above)
Comment (mrshirts) [reply _about_livecoms/02_thanks.md]: Cleaned up.

File: _about_livecoms/02_thanks.md (hunk 5 above)
Comment (mrshirts) [reply _about_livecoms/02_thanks.md]: Clean up and clarified.

File: _about_livecoms/03_earlycareer.md (hunk 2 above)
Comment (mrshirts) [reply _about_livecoms/03_earlycareer.md]: Done.

File: _about_livecoms/03_earlycareer.md (hunk 4 above)
Comment (mrshirts) [reply _about_livecoms/03_earlycareer.md]: Hmm. That doesn't really describe what it does for the _student_.  I adjusted a little bit here, and added more of this to the editor instructions.

File: _editorial_policies/02_editor_instructions.md [hunk 6]
```diff
@@ -30,7 +30,8 @@ The Lead Editor:
...
-- Identify suitable reviewers, who may include experts suggested by the authors, others in the field you already know of, or authors cited frequently in the article. LiveCoMS generally requires at least two standard reviewers in addition to a a student reviewer, though exceptional circumstances (such as extensive community feedback via GitHub) may result in exceptions.
+- Identify suitable reviewers, who may include experts suggested by the authors, others in the field you already know of, or authors cited frequently in the article. LiveCoMS generally requires at least two standard reviewers in addition to a a student reviewer, though exceptional circumstances (such as extensive community feedback via GitHub) may result in exceptions.   The requirement for a student reviewer for every submission is a unique strength of LiveCoMS that enhances the pedagogical value of every accepted paper.
```
Comment (dmzuckerman) [inline _editorial_policies/02_editor_instructions.md]: typo "a a student reviewer"

File: _about_livecoms/03_earlycareer.md [hunk 7]
```diff
@@ -2,18 +2,22 @@
...
 
 * If you are a graduate student or postdoc, we encourage you to sign
-  up to be a student reviewer. Please email
-  [the managing editors](mailto:managing@livecomsjournal.org)
+  up to be a student reviewer.  Every submission to LiveCoMS requires
+  a student reviewer to make sure that the material is accessible and
+  appropriate for all levels of users. By participating as a student
+  reviewer, you will help improve the pedagogical and practical value
+  of LiveCoMS to a diverse community of simulation scientsts.  Please
+  email [the managing editors](mailto:managing@livecomsjournal.org)
```
Comment (dmzuckerman) [inline _about_livecoms/03_earlycareer.md]: Good - better than what I suggested

**Comment by mrshirts** [comment]
I'm going to go ahead and merge since all raised points have been addressed.

**Pull request merged**